src/seedu/addressbook/commands/ListByCommand.java

```diff
@@ -0,0 +1,52 @@
+package seedu.addressbook.commands;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+/**
+ * Lists all persons in the address book to the user.
+ */
+public class ListByCommand extends Command {
+
+    public static final String COMMAND_WORD = "listBy";
```

Comment:
Commands are case sensitive, and this doesn't follow the precedence set by the `viewAll` command